src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="Images/logo.png"
+          alt="MoYo Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards__container">
+        <div
+          class="product__card"
+          data-qa="card"
+        >
+          <div class="product__image"></div>
+
+          <div class="product__info">
```

Comment:
Class names should be written in kebab-case. The class `product__info` should be `product-info`.